masters/models.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from django.db.models                import Model
+from django.db.models.deletion       import CASCADE
+from django.db.models.fields         import CharField, DateField, IntegerField, URLField
+from django.db.models.fields.related import ForeignKey, ManyToManyField
+
+from core.models import TimeStampModel
+
+EMAIL_REGEX    = r'^[a-zA-Z0-9]+@[a-zA-Z0-9]+.[a-zA-Z0-9]+$'
+PASSWORD_REGEX = r'^(?_.*[a-z])(?_.*[A-Z])(?_.*[0-9])(?_.*[!@#$%^&*()-=_+])[a-zA-Z0-9`~!@#$%^&*()_+-=;:,./<>?]{8,20}$'
+PHONE_REGEX    = r'^[0-9]{3}-[0-9]{3,4}-[0-9]{4}$'
+NAME_REGEX     = r'^[a-zA-Z가-힇]{2,10}$'
+
+class Master(TimeStampModel):
+    gender_choice = (
+        ('male', 'Male'),
+        ('female', 'Female')
+    )
+    kakao_id      = IntegerField()
+    name          = CharField(max_length=45)
+    phone         = CharField(max_length=20)
+    gender        = CharField(max_length=20, choices=gender_choice, null=True)
+    birth         = DateField()
+    regions       = ForeignKey('Region', on_delete=CASCADE)
+    profile_image = URLField()
+    main_service  = ForeignKey('services.Service', on_delete=CASCADE)
+    career        = IntegerField()
+    service       = ManyToManyField('services.Service', through='services.MasterService', related_name='masters')
```

Comment:
`master.services`로 객체 속성에 접근할 수 있도록 복수형으로 만들어주세요!
```suggestion
    services       = ManyToManyField('services.Service', through='services.MasterService', related_name='masters')
```

Reply:
수정하겠습니다!